### PR TITLE
[CARBONDATA-3053][Cli] Fix bugs for carbon-cli found in windows env

### PR DESCRIPTION
**Pull request by xuchuanyin**
Tests run failed in windows env due to:
1. unclosed file stream causing failure in clearing files in windows
2. the line separator is not "\n" in windows

In this commit, We fix the bugs.

Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [ ] Any interfaces changed?
 
 - [ ] Any backward compatibility impacted?
 
 - [ ] Document update required?

 - [ ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ ] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1332/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9384/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1123/

**Comment by xuchuanyin** [comment]
retest this please

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9387/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/1337/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/1127/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.1, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/9391/

File: tools/cli/src/main/java/org/apache/carbondata/tool/DataFile.java [hunk 1]
```diff
@@ -500,4 +503,7 @@ private double computePercentage(byte[] data, byte[] min, byte[] max, ColumnSche
     }
   }
 
+  public void close() throws IOException {
+    this.fileReader.finish();
```
Comment (jackylk) [inline tools/cli/src/main/java/org/apache/carbondata/tool/DataFile.java]: why not close?

Comment (xuchuanyin) [reply tools/cli/src/main/java/org/apache/carbondata/tool/DataFile.java]: this is the `close` method for `fileReader`

**Comment by jackylk** [comment]
LGTM

**Pull request closed without merging**